hivemind/p2p/p2p_daemon_bindings/p2pclient.py

```diff
@@ -0,0 +1,75 @@
+from typing import AsyncIterator, Iterable, Sequence, Tuple
```

Comment:
I suggest to make a docstring clarifying that this code comes from the [repo](https://github.com/mhchia/py-libp2p-daemon-bindings), mentioning the license and author name.

Same for every file in this directory.